src/ngraph/frontend/onnx_import/op/reduce.hpp

```diff
@@ -115,8 +120,12 @@ namespace ngraph
                 ///
                 inline NodeVector reduce_l2(const Node& node)
                 {
+                    auto reduce_function = std::bind(ngraph::builder::l2_norm,
```

Comment:
Please change the name from generic `reduce_function` to specific, like: `l2_norm_reduction`

Reply:
Doesn't this come from the name of the function this is in?